qpmodel/stmt.cs

```diff
@@ -671,15 +676,15 @@ public bool SubqueryIsWithMainQuery(NamedQuery subquery)
         {
             // FromQuery or decorrelated subqueries are merged with main plan
             var r = (fromQueries_.ContainsKey(subquery) ||
-                decorrelatedSubs_.Contains(subquery));
```

Comment:
are these changes from auto-format? please keep the indent space - the indent space meaning the new line  is not a new statement, but a continuation.

Reply:
I will run the auto-format.

Reply:
I think this line breaking is accidental because it is not part of normalization.

Reply:
I tried "format selection" but it did nothing. I will join the lines, is that alright?